rsocket-core/src/main/java/io/rsocket/RSocketFactory.java

```diff
@@ -99,9 +100,9 @@ static ServerRSocketFactory receive() {
   }
 
   interface MimeType<T> {
-    T mimeType(String dataMineType, String metadataMimeType);
+    T mimeType(String metadataMimeType, String dataMimeType);
```

Comment:
Even though it's good that switching the order of these two should be fixing the current users, as the users will not notice anything they can still assume that the first parameter is dataMimeType. 
So their code will still be 
```
mimeType("datamimetype", "metadatamimetype")
```
but in reality the first parameter is assigned to metadataMimeType now.

I mean, now we force them to have a bug without they notice. So I am in favor of not changing the order of these two.